containers/wordpress/wp_base/https/Dockerfile

```diff
@@ -0,0 +1,83 @@
+ARG baseimage=wp_base
+FROM $baseimage
+
+LABEL authors="nitin.tekchandani@intel.com,lin.a.yang@intel.com,uttam.c.pawar@intel.com"
+
+USER root
+
+RUN apt-get update -y && \
+    apt-get remove nginx -y && \
+    apt-get install -y \
+    libpcre3 \
+    libpcre3-dev
+
+
+# Build/Install openssl v1_1_1f
+RUN mkdir /home/${USERNAME}/temp
+WORKDIR /home/${USERNAME}/temp
+RUN git clone https://github.com/openssl/openssl.git && \
+    cd openssl && \
+    git checkout OpenSSL_1_1_1f && \
+    mkdir -p /home/${USERNAME}/openssl_install/lib/engines-1.1 && \
+    ./config --prefix=/home/${USERNAME}/openssl_install \
+             -Wl,-rpath=/home/${USERNAME}/openssl && \
+     make update && \
+     make -j 10 &&  \
+     make install
+WORKDIR /home/${USERNAME}
+RUN rm -rf /home/${USERNAME}/temp
+
+# Comment out RANDFILE entry in /etc/ssl/openssl.cnf
+RUN sed -iE 's/RANDFILE\(\s+\=\s\$ENV\:\:HOME\/\.rnd\)/#RANDFILE\1/' /etc/ssl/openssl.cnf
+
+# Create required certificates for https
+RUN mkdir -p /home/${USERNAME}/certificates/ssl
+WORKDIR /home/${USERNAME}/certificates
+RUN openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout server.key -out server.crt -subj "/C=US/ST=OR/L=IN/O=IN/OU=IN/CN=$(hostname)"
+RUN openssl ecparam -genkey -out key.pem -name prime256v1
+RUN openssl req -x509 -new -key key.pem -out cert.pem  -subj "/C=US/ST=OR/L=IN/O=IN/OU=IN/CN=$(hostname)"
+WORKDIR /home/${USERNAME}/certificates/ssl
+RUN openssl dhparam -out dhparam.pem 2048
+RUN chown -R ${USERNAME} /home/${USERNAME}/certificates
+
+# NGINX
```

Comment:
Suggest to use the NGNIX Async in the base. https://github.com/intel/asynch_mode_nginx.git

Reply:
This has been done in the latest commit.